app/models/equipment_model.rb

```diff
@@ -215,6 +215,16 @@ def available_count(date)
     total - reserved - number_overdue
   end
 
+  # figure out the qualitative status of this model's objects
+  def availability(date)
+    num = available_count(date)
+    total = equipment_objects.count
```

Comment:
Does this take into account situations when we have deactivated equipment items? You might have to call `equipment_objects.active.count` or something like that, I'm not sure.